src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsModelValidator.cs

```diff
@@ -71,5 +87,28 @@ public IEnumerable<ModelValidationResult> Validate(ModelValidationContext valida
 
             return Enumerable.Empty<ModelValidationResult>();
         }
+
+        private string GetErrorMessage(ModelValidationContextBase validationContext)
+        {
+            if (validationContext == null)
+            {
+                throw new ArgumentNullException(nameof(validationContext));
+            }
+            if (validationContext.ModelMetadata == null)
+            {
+                throw new ArgumentException(Resources.FormatPropertyOfTypeCannotBeNull(
+                     "ModelMetadata",
```

Comment:
Oh, I miss-remembered/understood some of @dougbu feedback, I thought he had told me not to do nameof on second level items, but he really only told me it's not an ArgumentNullException. Will fix unless @dougbu objects.